client/src/components/ResourceCategoryFilter.js

```diff
@@ -15,6 +15,19 @@ function ResourceCategoryFilter(props) {
     subcategorySelect,
   } = props;
 
+  const orderedCategories = {};
+  Object.keys(categories)
+    .sort()
+    .forEach(function(key) {
+      orderedCategories[key] = categories[key];
+      orderedCategories[key].sort(function(a, b) {
+        const textA = a.toUpperCase();
+        const textB = b.toUpperCase();
+        const bool = textA > textB ? 1 : 0;
+        return textA < textB ? -1 : bool;
+      });
+    });
+
```

Comment:
We can probably wrap this in a `useEffect` with `categories` as a dependency, so we aren't running this every time we render the component.  This would involve making `orderedCategories` a state variable

Reply:
wait what are variables a and b? can we make those clearer just for the sake of dumb people like me :)